erpnext/selling/doctype/quoting_sheet/quoting_sheet.js

```diff
@@ -42,14 +71,26 @@ frappe.ui.form.on("Quoting Sheet Item", {
 		if (row.qty && row.rate) {
 			let amount = row.qty * row.rate;
 			frappe.model.set_value(cdt, cdn, "amount", amount);
+			frm.trigger("calculate_total_raw_material_cost");
 		}
 	},
 
+	calculate_total_raw_material_cost: function (frm) {
+		frappe.call({
+			method: "calculate_total_raw_material_cost",
+			doc: frm.doc,
+			callback: () => {
+				frm.refresh_field("rm_cost");
```

Comment:
- Are we updating the value of rm_cost ?? If so I do not see a `set_value`

Reply:
@hrwX we are setting rm_cost value in python file, line no 44 and 45.
let me know if this not enough or something else.